main.go

```diff
@@ -144,31 +149,41 @@ func main() {
 		}
 	}
 
+	var xcconfigPath string
+	if cfg.XCConfigContent != "" {
```

Comment:
Not in this PR, but I think we should extract this logic to `go-xcode`, as it will be duplicated in `steps-xcode-test`. Wdyt?